### PR TITLE
Use console.dir instead of console.log for better object viewing

### PR DESCRIPTION
**Pull request by awmuncy**

### CODE REVIEW

File: src/commands/namespace.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
         mappings,
       };
 
-      console.log(response, {depth: null});
+      console.log(response, { depth: null });
```
Comment (carsonfarmer) [inline src/commands/namespace.ts:67]: Should this be `console.dir`?

**Review by carsonfarmer — CHANGES_REQUESTED** [review]

File: src/commands/namespace.ts (hunk 1 above)
Comment (awmuncy) [reply src/commands/namespace.ts]: Yes, fixed.

**Review by carsonfarmer — APPROVED** [review]
🤌

**Comment by awmuncy** [comment]
Note: Solved issue #201

**Pull request merged**